src/compile/compile.ts

```diff
@@ -8,15 +9,27 @@ import {assembleRootData} from './data/assemble';
 import {optimizeDataflow} from './data/optimize';
 import {Model} from './model';
 
+export interface CompileOption {
```

Comment:
CompileOptions

Reply:
Well, our config object isn't `Configs` either.

Reply:
Because it's one config with may options.

Reply:
You have a tendency to try to make things consistent when English is the most inconsistent language, lol.

Reply:
Whatever plurality is annoying anyway.